include/oneapi/dpl/internal/distributed_ranges_impl/detail/ranges.hpp

```diff
@@ -0,0 +1,272 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_DR_DETAIL_RANGES_HPP
+#define _ONEDPL_DR_DETAIL_RANGES_HPP
+
+#include <any>
+#include <iterator>
+#include <type_traits>
+
+#include "std_ranges_shim.hpp"
+
+namespace oneapi::dpl::experimental::dr
+{
+
+namespace ranges
+{
+
+template <typename>
+inline constexpr bool disable_rank = false;
```

Comment:
Is this a public customization point? In other words, do we allow the users to specialize this variable?